### ISSUE TITLE
General guide issues

### ISSUE THREAD

**Issue by evelinec** [OpenLiberty/draft-guide-sessions#2]
- [x] Propose to include complete URL for the POST request: /sessions/session/cart/{item}&{price}
=> This seems ok once open the UI, as the short URL is the value on the UI. 
=> Updated URL according to new change: /sessions/cart/{item}&{price}

- [x] Include complete URL for GET request to /sessions/session/cart
=> This seems ok once open the UI, as the short URL is the value on the UI. 
=> Updated URL according to new change: /sessions/cart

- [x]  "cart/session" => change symbol to word, ie, cart or session

- [x] "Client/Server" => Client or Server
- [ ]  The "sessionCache.png" image isn't showing up on the test site - pending on  https://github.com/OpenLiberty/openliberty.io/issues/935

Already done by Alex: 
- [x] The "Great work! You’re done!" section doesn't have any content

Review/test checklist: 
- [x] Working instruction and sample code
- [x] Quality of code (best coding practices)
- [x] Quality and presentation of guide according to Structure and Style Guideline
- [x] Consistency of guide with template and other guides
- [x] Examples with the right outcomes are provided
- [x] Example with the wrong outcomes, if any, are provided
- [x] Check that all licensing statements are properly stated in all files
- [x] Check that the pom.xml, server.xml, etc files are clean up
- [x] Check that the directories are properly structured
- [x] Test the guide on 3 platforms: Mac, Win, Liniux
- [ ] If any URL visits, try them on different browsers: Firefox, Chrome, Safari
- [ ] If any URL visits, try curl command where applicable 
- [x] Ensure that the guide is using the latest version of the liberty-maven-plugin or gradle plugin
- [x] Check that only these `page-tags` can be used in a guide (nothing else): `MicroProfile, Maven, Docker, Kubernetes, Gradle, Java EE`

- [ ] Testing with different IDEs, ie, Atom, VS.code, Eclipse, IntelliJ, (Microclimate) - New!
- [x] Run Acrolink Checker on draft (above 70 score approximately)
- [x] Consider SEO title and description for the guides 
- [x] Ensure automated test is enable, setup with Travis CI, and schedule tests 
- [x] Run `diff -r start/ finish/`
- [ ] Ensure that the automation tests are able to run with continuous daily build

**Comment by evelinec** [comment]
After refresh the environment, the tests are working.

**Comment by evelinec** [comment]
Tested the guide on MacOS and Linux.

**Comment by evelinec** [comment]
- [ ] Convert the guide to multipane

**Comment by evelinec** [comment]
- [ ] Run Acrolinx checker

**Comment by evelinec** [comment]
Completed for now.
